pkgs/applications/networking/ssb/patchwork/default.nix

```diff
@@ -23,7 +23,7 @@ let
     name = "ssb-patchwork";
     exec = "${binary}/bin/ssb-patchwork";
     icon = "ssb-patchwork.png";
-    comment = "Decentralized messaging and sharing app";
+    comment = "Client for the decentralized social network Secure Scuttlebutt";
```

Comment:
Changed this so that Patchwork can be found more easily via e.g. rofi by typing "scuttlebutt".